### PR TITLE
[Windows] Fixed VBS fallback

### PR DESCRIPTION
**Pull request by gpotter2**
This PR fixes the "VBS fallback" option when powershell is not available, which is currently broken.

This is due because the VBS parser works line by line, and when an object (like the IP) is not available on an interface, it would skip this line and break everything

It also adds tests and moves the MAC fix directly in the parser.

### CODE REVIEW

**Comment by gpotter2** [comment]
@guedou Hello, could you review that please ? (It's my current oldest PR)
Thanks !!

File: scapy/arch/windows/__init__.py [hunk 1]
```diff
@@ -225,6 +240,7 @@ def get_windows_if_list():
         query = exec_query(['Get-NetAdapter'],
                            ['InterfaceDescription', 'InterfaceIndex', 'Name',
                             'InterfaceGuid', 'MacAddress']) # It is normal that it is in this order
+
```
Comment (p-l-) [inline scapy/arch/windows/__init__.py]: Please avoid useless changes.

**Comment by gpotter2** [comment]
My bad. Fixed

**Comment by gpotter2** [comment]
@p-l- and that one should be ready to merge.

**Pull request merged**